src/main/java/uk/ac/ebi/subs/checklistservice/services/internal/archive/checklist/ena/EnaChecklistService.java

```diff
@@ -41,17 +42,14 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(EnaChecklistService.class);
 
-    private static final Map<String, String> CHECKLIST_NAME_LATEST_FILE_NAME_MAP = new HashMap<>(128);

```

Comment:
While testing, I realized that keeping this static makes this map survive the context refreshes during tests. Most likely because static objects are kept in the section of the memory that is in the scope of the class rather than the object of the class. Upon noticing this, I thought that, in essence, it really does not matter if this static or not because spring is only going to initialize this class only once.